config/formats.js

```diff
@@ -1414,6 +1414,118 @@ exports.Formats = [
 		},
 	},
 	{
+		name: "Type Illusion [Beta]",
+		section: "Exiled's Custom Gamemodes",
+		mod: 'gen7',
```

Comment:
There is no Gen 7 if you meant a gen 7 battle add [Gen 7] in front of the name like 

> name: "[Gen 7] Type Illusion [BETA]",